endpoint/contracts/examples/treasury/LockTreasury.sol

```diff
@@ -35,17 +35,35 @@ contract LockTreasury {
         string memory name = IERC20Metadata(token).name();
         string memory symbol = IERC20Metadata(token).symbol();
         bytes memory data = abi.encodeWithSelector(
-            bytes4(keccak256("mint(address,string,string,address,uint256)")),
+            bytes4(
+                keccak256("mint(address,string,string,address,uint256,uint256)")
+            ),
             token,
             name,
             symbol,
-            msg.sender,
-            amount
+            recv,
+            amount,
+            getChainId()
         );
-        IEndpoint(_endpoint).send(_rid, _rua, data);
+        IEndpoint(_endpoint).send(rid, rua, data);
+    }
+
+    function unlock(
+        address token,
+        uint256 amount,
+        address recv
+    ) external onlyEndpoint {
+        IERC20Metadata(token).safeTransfer(recv, amount);
```

Comment:
why `safeTransfer` can be called, but others need `endpoint.send`?

Reply:
this function is unlock, it will release the lock balance